sym_object.py

```diff
@@ -282,9 +281,34 @@ def delete(self):
                 self.resizeUIObject(parent, 1, 120 - parent.width)
 
         for child_name in self.connected_objects:
-            #self.state.sym_objects[child_name].delete()
+            # delete any connections from/to child
```

Comment:
why can't you use the deleteWire() function to do this?

Reply:
I tried using deleteWire() - which deletes from both the child and parent - but I ran into some issues where it couldn't find the child key from the function since we're only creating Wire objects from parent to child. So I thought it would be safer this way to delete from the dictionaries individually and then remove from the scene if it exists.

Reply:
ok cool just curious